brendenwest/cascade
# About screen

## Description
closes #34 

This feature updates the template 'About Screen" with actual data.
The About screen will now contain:
- A scrollable view to read about the bike club.
- An social media element _outside_ of the scrollable view that contains links to all cascade's social media links.
- In the social media element there is also a JOIN, DONATE, and SHOP button added that links to their website.

## To Test
1. Pull the branch
2. Run the metro 
3. Run the app
4. Click on the hamburger drawer menu button (top left)
5. Click about
6. Verify the page loads correctly
7. Verify the scrollable view is usable
8. Verify ALL the buttons work

### Screenshots
<img src="https://user-images.githubusercontent.com/35348015/140633675-70c7c128-95b0-4774-b8f1-d7e43d39baf1.png" width="450">
